### PR TITLE
fix: tests failing in headful

### PR DESCRIPTION
**Pull request by OrKoN**
A collection of fixes for the tests failing in headful. There are three failing tests remaining after this PR.

### CODE REVIEW

File: src/node/Launcher.ts [hunk 1]
```diff
@@ -203,6 +203,7 @@ class ChromeLauncher implements ProductLauncher {
 
   defaultArgs(options: BrowserLaunchArgumentOptions = {}): string[] {
     const chromeArguments = [
+      '--allow-pre-commit-input', // TODO(crbug.com/1320996): neither headful nor headless should rely on this flag.
```
Comment (ergunsh) [inline src/node/Launcher.ts:206]: Can we add this argument only for the related tests? (or is it needed in all tests?)

**Review by ergunsh — APPROVED** [review]
LGTM

File: src/node/Launcher.ts (hunk 1 above)
Comment (OrKoN) [reply src/node/Launcher.ts:206]: It's also needed for runtime (not only tests) in headful unfortunately

Comment (ergunsh) [reply src/node/Launcher.ts:206]: Oh understood, thank you!

**Pull request merged**